tests/datamodules/test_deepglobelandcover.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'm trying to get rid of all of the files in `tests/datamodules` and instead test them directly with a trainer. Can you delete this file and add a new line in `tests/trainers/test_segmentation.py` instead? These `tests/datamodules` tests don't actually ensure that the datamodule works properly.

Reply:
This is needed to get test coverage on the val_pct argument to the datamodule (similar to other tests in the datamodules directory).

Reply:
(Although I added it to the segmentation trainer test list too)

Reply:
Can we add two `tests/conf/deepglobelandcover_*.yaml` files, one for val_pct = 0 and one for 0.5? That's what I did for other datasets.